json_rpc/private/jrpc_conv.nim

```diff
@@ -0,0 +1,94 @@
+# json-rpc
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+import
+  std/[tables, macros],
+  results,
+  json_serialization,
+  json_serialization/stew/results as jser_results,
+  json_serialization/std/[options, sets, tables]
```

Comment:
why are we import/exporting these? it seems like an application choice?

Reply:
removed